tests/server/extensions/test_beacon_extension.py

```diff
@@ -27,6 +27,24 @@ def test_beacon_create_app():
         assert beacon.token
 
 
+@responses.activate
+def test_add_dataset(app, institute_obj):
+    """Test the function that sends a POST request to the Beacon with the info to create a new dataset"""
+
+    # GIVEN a mocked Beacon server dataset add endpoint
+    url = "http://localhost:6000/apiv1.0/add_dataset"
```

Comment:
A fixture (or a constant) for at least the base URL would make it easier to read/maintain!

Reply:
Good idea, Added it in a constant!